web/src/main/java/com/graphhopper/http/GraphHopperApplication.java

```diff
@@ -45,9 +45,11 @@ public void initialize(Bootstrap<GraphHopperServerConfiguration> bootstrap) {
         bootstrap.addCommand(new MatchCommand());
 
         Map<String, String> resourceToURIMappings = new HashMap<>();
-        resourceToURIMappings.put("/assets/", "/maps/");
+        resourceToURIMappings.put("/com/graphhopper/maps/assets", "/maps/");
         resourceToURIMappings.put("/META-INF/resources/webjars", "/webjars"); // https://www.webjars.org/documentation#dropwizard
         bootstrap.addBundle(new ConfiguredAssetsBundle(resourceToURIMappings, "index.html"));
+        // todonow: shall we remove configured assets bundle (and use dropwizard-assets instead)?
+//        bootstrap.addBundle(new AssetsBundle("/com/graphhopper/maps/assets/", "/maps/", "index.html"));
```

Comment:
I was hoping to fix the problem that pt and isochrone require a tailing slash, but using AssetsBundle instead of ConfiguredAssetsBundle did not help unfortunately. Still, why do we need configured assets bundle? When we start the server from the IDE we can still change the code without creating a new jar.

Reply:
Did you try to change something in JavaScript and did this change appear with `npm watch` & refresh? I can remember that there was a problem with this.

Reply:
Yes: https://github.com/graphhopper/graphhopper/pull/2238
But I could not make it work for src/main/resources/assets and if that does not work I see no reason to keep configured-assets-bundle.

Reply:
Ah no wait. It works for index.html even for src/main/resources/assets. The reason I thought it does not work is that the javascript code is copied to target/classes/assets (using `npm run watch`)... The 'override' is not really an override because dropwizard still reads from target/classes/assets (by default).

Reply:
I think we should remove it as in #2238. I have already merged #2238 into this PR.

Reply:
See #2238